models/capital_v2.json

```diff
@@ -157,5 +157,28 @@
             "column": "amount",
             "label": "Amount"
         }
+    },
+    "can_aggregate": "True",
+    "aggregate_columns": {
+        "new_assets": {
+            "column": "DEPRECIATION",
+            "label": "Depreciation"
+        },
+        "renewal_of_existing": {
+            "column": "NEW",
+            "label": "New"
+        },
+        "total_assets": {
+            "column": "RENEWAL",
+            "label": "Renewal"
+        },
+        "repairs_maintenance": {
+            "column": "REPAIR_MNT",
+            "label": "Repairs and maintenance"
+        },
+        "asset_register_summary": {
+            "column": "UPGRADING",
+            "label": "Upgrading"
+        }
```

Comment:
this seems to be hardcoding what's already available here https://municipaldata.treasury.gov.za/api/cubes/capital_v2/members/capital_type

One way to access these labels for presentation would be to perform that query. Another way would be to include the label as a drilldown as well, similarly to how we do item code and label.

Reply:
Now fetching the labels from that endpoint. Where` this.renderValues` is being called, the `this` object  does not exist in that function, I'm not sure why it's doing that so I just parsed the `aggregate_columns` variable directly